out-adapter-persistence-jpa/src/main/java/gohigher/user/entity/RefreshTokenRepository.java

```diff
@@ -0,0 +1,20 @@
+package gohigher.user.entity;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RefreshTokenRepository extends JpaRepository<RefreshTokenJpaEntity, Long> {
+
+	@Modifying(flushAutomatically = true, clearAutomatically = true)
+	@Query("UPDATE RefreshTokenJpaEntity r SET r.value = :value WHERE r.userId = :userId")
+	void updateValueByUserId(Long userId, String value);
+
+	Optional<RefreshTokenJpaEntity> findByUserId(Long userId);
+
+	@Modifying(flushAutomatically = true, clearAutomatically = true)
+	@Query("DELETE FROM RefreshTokenJpaEntity r WHERE r.userId = :userId")
+	void deleteByUserId(Long userId);
```

Comment:
```suggestion
	void deleteByUserId(Long userId);
```
Repository 테스트 코드 상에서는 위의 어노테이션 없이도 테스트가 성공합니다! 혹시 해당 어노테이션들을 추가한 이유가 있으실까요??

Reply:
delete와 관련해서 SimpleJpaRepository가 조회 후 삭제라는 로직이 발생하는 것으로 알고 있어요! 그래서 쿼리를 한번으로 줄이기 위해 사용했습니다!